### ISSUE TITLE
Aggregate failling.

### ISSUE THREAD

**Issue by fulmicoton** [gz/autoperf#2]
**Describe the bug**

When running autoperf aggreate I get the following error

```
[2019-05-02T03:15:18Z ERROR autoperf::aggregate] "./out/10_stat.csv": Event 'MEM_LOAD_RETIRED.L1_MISS' was not measured correctly with perf. This is a bug, please report it!
[2019-05-02T03:15:18Z ERROR autoperf::aggregate] "./out/14_stat.csv": Event 'FRONTEND_RETIRED.LATENCY_GE_512' was not measured correctly with perf. This is a bug, please report it!
[2019-05-02T03:15:18Z ERROR autoperf::aggregate] "./out/24_stat.csv": Event 'BR_INST_RETIRED.FAR_BRANCH' was not measured correctly with perf. This is a bug, please report it!
[2019-05-02T03:15:18Z ERROR autoperf::aggregate] "./out/35_stat.csv": Event 'MEM_LOAD_RETIRED.L3_HIT' was not measured correctly with perf. This is a bug, please report it!
...
```

**Machine (please complete the following information):**
-Linux version: Linux paul-laptop 4.18.0-16-generic #17-Ubuntu SMP Fri Feb 8 00:06:57 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux
-perf version: perf version 4.18.20
-autoperf version: autoperf 0.2
-output of: breakpoint cstate_core i915 kprobe power tracepoint uncore_cbox_0 uncore_cbox_2 uprobe cpu cstate_pkg intel_pt msr software uncore_arb uncore_cbox_1 uncore_cbox_3
 - Machine: attached
 - output of: ls /sys/bus/event_source/devices/
```
breakpoint  cstate_core  i915      kprobe  power     tracepoint  uncore_cbox_0  uncore_cbox_2  uprobe
cpu         cstate_pkg   intel_pt  msr     software  uncore_arb  uncore_cbox_1  uncore_cbox_3
```

**Comment by gz** [comment]
Thanks I will look into this, there is an issue with measuring some of these latency events. The aggregate command should still have worked btw, it will just remove these events from the final results.csv data set.

**Comment by gz** [comment]
I think this should be fixed in 0.9.4 let me know if the error still appears.
